build_tools/mako/configuration.py

```diff
@@ -176,7 +176,10 @@ def get_s20_default_target_list(skipped_target=None, batch_config=None):
               "--iree-vulkan-target-triple=valhall-g77-unknown-android10",
               "--iree-flow-dispatch-linalg-on-tensors",
               "--iree-codegen-spirv-experimental-linalg-on-tensors",
-              "-iree-flow-inline-constants-max-byte-length=2048"
+              "-iree-flow-inline-constants-max-byte-length=2048",
+              "-iree-spirv-enable-vectorization",
+              # TODO(GH-5330): Remove the flag.
+              "-iree-flow-inline-constants-max-byte-length=16"
```

Comment:
This is duplicated with L179. We should delete 179.

Reply:
ah, good catch...